Improve behaviour of option -W (#933)
twisted/pydoctor: README.rst

in development
^^^^^^^^^^^^^^

> * When option ``-W/--warnings-as-errors`` is used, all warnings issued with the warnings module now makes pydoctor exit with code 3.

Reviewer: It's actually only warnings that pydoctor issues, not "all warnings issued with the warnings module". Perhaps:

```suggestion
* When option ``-W/--warnings-as-errors`` is used, pydoctor exits with code 3 when it issues any warning.
```
